.github/ISSUE_TEMPLATE/source-connector-type.md

```diff
@@ -0,0 +1,26 @@
+---
+
+name: Source Connector Type
+about: Add a new type or update an existing type in source connector
```

Comment:
this is for adding support for new types in the Airbyte type system? what's an example for how it would be used?

Reply:
This is what we talked about on Monday. It is not just adding new types, but also for updating an existing type. An example is to change the timestamp precision from second to millisecond.